website/docs/d/pi_virtual_serial_number.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: ibm_pi_virtual_serial_number"
+description: |-
+  Provides data for a virtual_serial_number in an IBM Power Virtual Server cloud.
+---
+
+# ibm_virtual_serial_number
+Retrieve information about an existing IBM Power Virtual Server Cloud Instance as a read-only data source. For more information, about IBM power virtual server cloud, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
```

Comment:
The description need to updated, we are retrieving info about `virtual serial number` not a `IBM Power Virtual Server Cloud Instance`
we will have to update the url, later on.